app/services/view/AutoCompleteItem.scala

```diff
@@ -16,14 +16,16 @@
 
 package services.view
 
-import models.codes.ProcedureCode
+import models.codes.{AdditionalProcedureCode, ProcedureCode}
 import services.model.{Country, CustomsOffice, OfficeOfExit, PackageType}
 import services.{DocumentType, HolderOfAuthorisationCode}
 
 case class AutoCompleteItem(label: String, value: String)
 
 object AutoCompleteItem {
 
+  def formatProcedureCode(pc: ProcedureCode) = s"${pc.code} - ${pc.description}"
```

Comment:
Personal opinion :smiling_face_with_three_hearts: 
`formatProcedureCode` should be in `ProcedureCodeService`.

Reply:
I strongly disagree. This format is a UI level responsibility, not a data retrieval level one.

Reply:
But `ProcedureCodeService` is not a "data retrieval" class. It's a **service** class only used for UI purpose. And in fact it should be then placed under /services/view.